calico_containers/tests/st/workload.py

```diff
@@ -44,9 +48,8 @@ def __init__(self, host, name, ip=None, image="busybox", network=None):
         ]
         assert ip is None, "Static IP assignment not supported by libnetwork."
         if network:
-            assert isinstance(network, DockerNetwork)
-            # Using @squaremo's Docker UI patch, --net accepts
-            # <driver name>:<network name> as an option.
+            if network is not "none":
+                assert isinstance(network, DockerNetwork)
```

Comment:
A cleaner way to accomplish this would be to create module constants `NONE`, `BRIDGE` etc., for each of the built-in reserved networks that are instances of DockerNetwork.  Then we can avoid passing strings around as constants which makes maintenance a pain.